app/assets/javascripts/helpers/action_click.js

```diff
@@ -0,0 +1,26 @@
+(function(App) {
```

Comment:
I don't think a helper for click actions would be useful for what we'll be using it for, what do you think if we just add a listener in the Share Modal, to bind a global click event to a specific class in this case for the footer link?